tests/commands/push.ts

```diff
@@ -1,11 +1,11 @@
-import { CLASP, TEST_CODE_JS } from '../constants';
-import { cleanup, setup } from '../functions';
-import { describe, it } from 'mocha';
-
+import { spawnSync } from 'child_process';
 import { expect } from 'chai';
 import fs from 'fs-extra';
-import { spawnSync } from 'child_process';
+import { describe, it } from 'mocha';
+import { CLASP, TEST_CODE_JS } from '../constants';
+import { cleanup, setup } from '../functions';
 
+// TODO: add some `.html` files to tests
```

Comment:
Can we add a test so we don’t break anything in the future? I can’t publish this till next week.